src/commands/focus/focusGroup.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { hasFocusedGroupContextKey } from "../../constants";
+import { ext } from "../../extensionVariables";
+import { GroupingItem } from "../../tree/azure/GroupingItem";
+
+export async function focusGroup(context: IActionContext, item: GroupingItem): Promise<void> {
+    if (item.resourceGroup) {
+        ext.focusedGroup = {
+            kind: 'resourceGroup',
+            id: item.resourceGroup.id.toLowerCase(),
+        }
+    } else if (item.resourceType) {
+        ext.focusedGroup = {
+            kind: 'resourceType',
+            type: item.resourceType,
+        }
+    } else if (item.location) {
+        ext.focusedGroup = {
+            kind: 'location',
+            location: item.location,
+        }
+    }
```

Comment:
No ARM tag group support?

Reply:
For now I'm not adding ARM tag group support since it requires more refactoring than I'd like to do at this moment. The focus command will not appear on ARM tag groups.